packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -91,16 +91,20 @@ export class MetamaskPage implements WalletPage {
   async switchNetwork(networkName = 'Linea Mainnet') {
     await test.step(`Switch network to "${networkName}"`, async () => {
       await this.navigate();
-      await this.header.networkSetting.switchNetwork(networkName);
+      await this.networkList.switchNetwork(networkName);
```

Comment:
is it work now? Because Metamask wallet was updated las week and flow for select network was changed.

Reply:
work